src/iocore/cache/RamCacheLRU.cc

```diff
@@ -84,13 +85,14 @@ RamCacheLRU::size() const
 
 ClassAllocator<RamCacheLRUEntry> ramCacheLRUEntryAllocator("RamCacheLRUEntry");
 
-static const int bucket_sizes[] = {127,     251,      509,      1021,     2039,      4093,      8191,     16381,
-                                   32749,   65521,    131071,   262139,   524287,    1048573,   2097143,  4194301,
-                                   8388593, 16777213, 33554393, 67108859, 134217689, 268435399, 536870909};
+static const int bucket_sizes[] = {8191,    16381,   32749,    65521,    131071,   262139,    524287,    1048573,   2097143,
+                                   4194301, 8388593, 16777213, 33554393, 67108859, 134217689, 268435399, 536870909, 1073741827};
 
 void
 RamCacheLRU::resize_hashtable()
 {
+  ink_release_assert(ibuckets < static_cast<int>(sizeof(bucket_sizes)));
```

Comment:
Is this correct? `sizeof(bucket_sizes)` is the number of bytes, which means it is `sizeof(int)` * number of entries, not the number of entries.

Reply:
Fixed